.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: ruby
+rvm:
+- 2.3.3
+
+# Build all branches, *including* gh-pages:
+branches:
+  only:
+  - gh-pages
+  - /.*/
+
+script: bundler exec jekyll build
+
+# route build to the container-based infrastructure for a faster build
+sudo: false
```

Comment:
This is the default now, no?

Reply:
I don't know. I copied that bit from the Jekyll documentation.

Reply:
Doesn't matter. Better to be explicit.